dandi/cli/command.py

```diff
@@ -9,11 +9,13 @@
 
 import click
 from click_didyoumean import DYMGroup
+import girder_client as gcl
```

Comment:
I think so far we did manage to avoid importing it in the path of CLI invocation so e.g. during `dandi --help`... please move it to the point of use -- seems to save 50-100ms from run time.  Minor -- but it all adds up

Reply:
Done.